package.json

```diff
@@ -5,13 +5,12 @@
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "webpack-dev-server --config ./webpack/dev/webpack.dev.config.js --content-base --public --hot --host 0.0.0.0 --disable-host-check",
-    "dev": "webpack-dev-server --config ./webpack/dev/webpack.dev.config.js --content-base --public --host 0.0.0.0 --disable-host-check",
-    "build": "npm run clean && webpack --config ./webpack/prod/webpack.prod.config.js",
+    "start": "npm run build && serve --single ./public",
```

Comment:
This is actually not a good idea. Now.sh runs `npm run build` automatically before running the start command. With this configuration, the deployment will take twice as long.

Reply:
Done! Add `now-start` to serve only.